src/skins/vector/css/common.css

```diff
@@ -161,8 +167,8 @@ input[type=text]:focus, textarea:focus {
     left: 0;
     width: 100%;
     height: 100%;
-    background-color: #000;
-    opacity: 0.2;
+    background-color: #e9e9e9;
```

Comment:
this looks like it will break lightbox background color?  `.mx_Dialog_lightbox` presumably now needs an explicit `color: #000;`